test/params_test.c

```diff
@@ -306,7 +301,6 @@ static BIGNUM *app_p3 = NULL;         /* "p3" */
 static unsigned char bignumbin[4096]; /* "p3" */
 static char app_p4[256];              /* "p4" */
 static char app_p5[256];              /* "p5" */
-static const char *app_p6 = NULL;     /* "p6" */
```

Comment:
Please restore, but make it an array that initialized with app_p6_init.

Reply:
I don't see how this helps, the pointer inside the OSSL_PARAM is being redirected.  The buffer remains untouched.